client/src/data_layer_configs/population_by_census_block_config.ts

```diff
@@ -104,7 +104,7 @@ const legendInterpolation = [
 
 const legendInterpolationCounty = [
```

Comment:
nit: Should this variable name change to say zip as well?

Reply:
Good catch